### DISCUSSION TITLE
EAN barcode generator

### DISCUSSION THREAD

**Discussion by ghost** [djsime1/awesome-flipperzero#57] [Ideas]
Although I welcome the introduction of a UPC-A barcode generator to the Unleashed firmware, for someone living in Europe (the EU) - like Me - it is mostly useless. I would pretty much like if there was a barcode generator using the European standard EAN format added.
Anyone in favor of that(?), pls like, comment and support. THX

**Comment by djsime1** [comment]
This repo is focused on the organization and listing of existing apps, not so much creating new ones. However, I'm sure someone could fork the UPC barcode generator app and rework it to support generation of EAN codes. If that comes along, then I'll gladly add it to the list.
